app/services/bills/fetch-bill-service.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+/**
+ * Fetches data needed for the bill page which includes a summary for each licence linked to the bill
+ * @module FetchBillService
+ */
+
+const BillModel = require('../../models/water/bill.model.js')
+const { db } = require('../../../db/db.js')
+
+/**
+ * Fetch the matching Bill plus a summary for each licence linked to it
+ *
+ * Was built to provide the data needed for the '/bills/{id}' page
+ *
+ * @param {string} id The UUID for the bill to fetch
+ *
+ * @returns {Object} the matching instance of BillModel plus a summary (ID, reference, and total net amount) for each
+ * licence linked to the bill
+ */
+async function go (id) {
+  const bill = await _fetch(id)
+  const licenceSummaries = await _fetchLicenceSummaries(id)
+
+  return {
+    bill,
+    licenceSummaries
+  }
+}
+
+async function _fetch (id) {
```

Comment:
I'd probably rename this function to `_fetchBill` as there are 2 fetches going on here. Just looks a bit inconsistent with the other function being called `_fetchLicenceSummaries`.

Up to you, you're the boss 😉